app/gui/src/dashboard/layouts/AssetContextMenu.tsx

```diff
@@ -248,19 +251,31 @@ export default function AssetContextMenu(props: AssetContextMenuProps) {
               }}
             />
           )}
-        {asset.type === backendModule.AssetType.project && isCloud && (
+        {asset.type === backendModule.AssetType.project && isCloud && enableHybridExecution && (
           <ContextMenuEntry
             hidden={hidden}
             action="run"
             isDisabled={!canOpenProjects}
             tooltip={disabledTooltip}
-            doAction={() => {
-              openProjectMutation.mutate({
-                id: asset.id,
-                title: asset.title,
-                parentId: asset.parentId,
-                type: state.backend.type,
-                inBackground: true,
+            doAction={async () => {
+              invariant(localBackend != null, 'Local Backend is null')
```

Comment:
Invariant is supposed for `uh-oh` situations, where we did something really bad. It means we can't _recover_ from this error and will display a full screen error boundary. Not sure it's a good idea here. Better to hide this menu if `local backend` is not available.